python/main.py

```diff
@@ -0,0 +1,34 @@
+import time
+
+import xpring
```

Comment:
Where does `xpring` come from?

Reply:
I publish this package to PyPI: https://pypi.org/project/xpring/

Reply:
It is a Python implementation of serialization, signing, and gRPC for XRPL. No calls to JavaScript.